### PR TITLE
Add Header and Footer

### PR DESCRIPTION
**Pull request by bernhardgr42**
Adding Header and Footer

### CODE REVIEW

File: vocabularies/PDF.xml [hunk 1]
```diff
@@ -80,6 +80,12 @@
             <String>Is padding (left, right, bottom, top) supported?</String>
           </Annotation>
         </Property>
+        <Property Name="HeaderFooter" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Header and Footer of document" />
```
Comment (HeikoTheissen) [inline vocabularies/PDF.xml:84]:
```suggestion
          <Annotation Term="Core.Description" String="Whether the document contains Header and Footer" />
```
This boolean property only controls whether header and footer exist, not how they look, right?

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: vocabularies/PDF.xml (hunk 1 above)
Comment (bernhardgr42) [reply vocabularies/PDF.xml:84]: Hi Heiko,

this annotation tells the client if the backend is capable to add a header or a footer. A requirement is that the "PDF API" is able to do this and this is only supported with a certain Backend version.

To control the page number, data/time stuff in a header or footer the client has to do a Create request on 
<EntityType Name="DocumentDescription"> of our common service in OData.

Comment (HeikoTheissen) [reply vocabularies/PDF.xml:84]: Will you then rephrase the description as I suggested?

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — DISMISSED** [review]

File: vocabularies/PDF.xml [hunk 2]
```diff
@@ -80,6 +80,12 @@
             <String>Is padding (left, right, bottom, top) supported?</String>
           </Annotation>
         </Property>
+        <Property Name="HeaderFooter" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="A header and Footer of the document is supported" />
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:84]:
```suggestion
          <Annotation Term="Core.Description" String="Page header and footer supported" />
```

**Review by ralfhandl — COMMENTED** [review]
Rephrase description

**Review by ralfhandl — DISMISSED** [review]

**Review by HeikoTheissen — APPROVED** [review]

File: vocabularies/PDF.xml [hunk 3]
```diff
@@ -80,6 +80,12 @@
             <String>Is padding (left, right, bottom, top) supported?</String>
           </Annotation>
         </Property>
+        <Property Name="HeaderFooter" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
```
Comment (ralfhandl) [inline vocabularies/PDF.xml:83]:
```suggestion
        <Property Name="HeaderFooter" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
          <Annotation Term="Common.Experimental" />
```

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**